src/pages/index.astro

```diff
@@ -3,60 +3,79 @@ import Layout from "../layouts/Layout.astro";
 import CardProject from "../components/CardProject.astro";
 import PageHeader from "../components/PageHeader.astro";
 import Modal from "../components/Modal.astro";
-const baseSkills = ["HTML", "CSS", "JavaScript"];
+// const baseSkills = ["HTML", "CSS", "JavaScript"];
+import { Octokit } from "@octokit/rest";
 
-const weatherCard = {
-  img: {
-    src: `${import.meta.env.BASE_URL}/project_1.png`,
-    alt: "Weather App Layout",
-  },
-  title: "Weather App",
-  description:
-    "Stay ahead of the forecast with our intuitive weather app. Get real-time updates on current conditions, hourly forecasts, and daily outlooks. Plan your day confidently with accurate weather information at your fingertips. Never be caught off guard by the elements again!",
-  skills: baseSkills,
-};
+const octokit = new Octokit({
+  auth: import.meta.env.OCTOKIT_TOKEN,
+  userAgent: "My Portfolio",
+});
 
-const calculatorCard = {
-  img: {
-    src: `${import.meta.env.BASE_URL}/project_2.png`,
-    alt: "Age calculator Layout",
-  },
-  title: "Age calculator",
-  description:
-    "AgeEase: Instantly calculate your age or anyone else' s with precision and simplicity. Whether you're celebrating milestones or planning events, keeps track of the years, months, and days. Effortlessly determine ages for birthdays, anniversaries, or any special occasion.",
-  skills: baseSkills,
-};
+const repos = await octokit.request("GET /users/{username}/repos", {
+  accept: "application/vnd.github+json",
+  username: "FrostEvil",
+  sort: "created",
+  per_page: 5,
+});
 
-const realEstateCard = {
-  img: {
-    src: `${import.meta.env.BASE_URL}/project_3.png`,
-    alt: "Real Estate welcome page",
-  },
+// const titles = repos.data.map((data) => Object.values(data["name"]).join(""));
 
-  title: "Real Estate",
-  description:
-    "Introducing Real Estate: Your trusted partner in real estate management. From residential complexes to commercial spaces, we offer comprehensive property management services tailored to meet your needs. With a focus on efficiency, transparency, and maximizing property value.",
-  skills: baseSkills,
-};
+const titles = repos.data.map((data) => {
+  return data["name"];
+});
```

Comment:
Does it look similar to the map we do in the template?

Reply:
Yes, I changed map in the template and used repos.data there.